Sources/RealDeviceMap/Webhook/WebHookRequestHandler.swift

```diff
@@ -63,7 +63,12 @@ class WebHookRequestHandler {
                 return response.respondWithError(status: .unauthorized)
             }
 
-            let loginSecretHeader = request.header(.authorization)
+            var loginSecretHeader = request.header(.authorization)
+            if let madAuth = Data(base64Encoded: loginSecretHeader?.components(separatedBy: " ").last ?? ""),
```

Comment:
Is there a more elegant way to check if it's mad rather than just trying it for each device?
Do they set a header that shows that it's from a MAD device?

Reply:
Not sure what RDM's providers' headers look like, but PogoDroid does send an "Origin" along in the header. That could be an indicator/difference ;)

Reply:
Thank-you for the suggestion Grennith, I've implemented it.